plugins/inputs/gnmi/README.md

```diff
@@ -70,6 +70,14 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   # prefix = ""
   # target = ""
 
+  # Support Juniper Extension 
+  # Some Sensors are directly manages by Linecard. In this case Linecard adds Juniper GNMI Extension Header
+  # By setting up the jnpr_extension flag: this allows the decoding of the Extension header if present
+  # Additionnal tags are directly extracted for this extension header
+  # Tags are: component_id, component_name, sub_component_id
+  # These tags are useful for aggreation 
+  # jnpr_extension = false
```

Comment:
In general we prefer to avoid adding lots of boolean options. I think this is the first time we have a vendor specific item as well. What do you think about having a vendor specific string array that defines specific vendor options. Something like the following and a user would enable this with `vendor_specific = ["jnpr_extension"]`

```suggestion
  ## Vendor specific options
  ## This defines what vendor specific options to load. 
  ## * Juniper Extension (jnpr_extension): some sensors are directly managed by
  ##   Linecard, which adds the Juniper GNMI Extension Header. Enabling this
  ##   allows the decoding of the Extension header if present.
  # vendor_specific = []
```

I would prefer we avoid needing the boolContainer you have below as well.